api/src/templates/mail/invited-to-platform.txt

```diff
@@ -1,6 +1,6 @@
 Hi {{invite.email}},
 
-You've been invited to join '{{invite.project.name}}' on Traduora with the role '{{user.role}}'.
+You've been invited to join '{{invite.project.name}}' on Traduora with the role '{{role}}'.
```

Comment:
The project name is now displayed, but the role is still missing, you can fix that by switching to `{{invite.role}}` instead of `{{role}}` :)